actioncable/lib/action_cable/server/configuration.rb

```diff
@@ -7,7 +7,7 @@ class Configuration
       attr_accessor :connection_class, :worker_pool_size
       attr_accessor :channel_load_paths
       attr_accessor :disable_request_forgery_protection, :allowed_request_origins
-      attr_accessor :cable, :url
+      attr_accessor :cable, :url, :mounted
```

Comment:
If we remove `action_cable_meta_tag`, then `url` could nominally go away. Let's merge `url` and `mounted` into a single `mount_path`, which defaults to `/cable`, and can be set to `nil` (do not mount), or any other [server-local] mount point.